packages/machine-extractor/src/sky/skyConfigModifySource.ts

```diff
@@ -66,13 +66,9 @@ export const modifySkyConfigSource = async (opts: { filePath: string }) => {
     });
 
     const output = recast.print(ast).code;
-    const prettierConfig = await prettier.resolveConfig(opts.filePath);
```

Comment:
This could create a version conflict with Prettier where it would use the user's Prettier config, which might not be compatible with the bundled Prettier.

Reply:
Would the problem get largely addressed if we just wouldn't bundle all of ur deps? I noticed this just yesterday and I think that we shouldn't be using `--bundle` like this here:
https://github.com/statelyai/xstate-tools/blob/0495b7489b871b10ddbcf3a31218e2d8bcdc649d/apps/cli/package.json#L11

Reply:
Yes, that seems to work as well. Thanks for the feedback. I've updated the build script in the latest commit.

Reply:
Wait, that didn't entirely work after a restart of my IDE. So, I'll remove the Prettier config part again but keep it unbundled.

Reply:
Going back to the bundled version again. I couldn't get it to not fail with the unbundled version. So maybe the unbundling is something we can handle in another PR?

Reply:
> So maybe the unbundling is something we can handle in another PR?

Yes, let's leave it out.

> This could create a version conflict with Prettier where it would use the user's Prettier config, which might not be compatible with the bundled Prettier.

That's why we try to honor the local version of Prettier in other places where we use it:
https://github.com/statelyai/xstate-tools/blob/2c8387fa9fc9ad26817cf90a7b9d54067fa8395c/apps/cli/src/typegen/writeToFiles.ts#L26

I think that a good chunk of `modifySkyConfigSource` doesn't belong to the `machine-extractor` package. If you'd move the FS interaction to `@xstate/cli` then you'd have an easier time using Prettier correctly there.

Reply:
Thanks, I moved to use the same `getPrettierInstance` function and did a mini-refactor of `modifySkyConfigSource `

Reply:
note that this is a breaking change - but since sky is experimental right now, I'm OK with this and I think that the refactor that was made here is an improvement 👍